hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -134,57 +142,59 @@
       tests: ["i2c_fifo_watermark"]
     }
     {
-      name: fifo_overflow
+      name: host_fifo_overflow
       desc: '''
-            Test the overflow interrupt of fmt_fifo and rx_fifo.
+            Test the overflow interrupt for fmt_fifo and rx_fifo.
 
             Stimulus:
-              - Keep sending a number of format byte higher than fmt_fifo and rx_fifo depth
+              - Configure DUT/Agent to Host/Target mode respectively      
+              - DUT keeps sending a number of format byte higher than the size of fmt_fifo and rx_fifo depth
 
             Checking:
               - Ensure excess format bytes are dropped
               - Ensure fmt_overflow and rx_overflow interrupt are asserted
-              - Ensure receving correct number of fmt_fifo and rx_fifo watermark interrupts
             '''
       milestone: V2
       tests: ["i2c_fifo_overflow"]
     }
     {
-      name: fmt_reset
+      name: target_fifo_empty,
       desc: '''
-            Test fmt_fifo reset.
+            Test tx_empty and tx_nonempty interrupt.
 
             Stimulus:
-              - Fill up the fmt fifo with data to be sent out
-              - Reset the fmt_fifo randomly after a number of bytes shows up on fmt_fifo 
+              - Configure DUT/Agent to Target/Host mode respectively
+              - Agent sends transaction to the DUT
 
             Checking:
-              - Ensure the remaining entries are not show up after fmt_fifo is reset
+              - Ensure tx_empty/tx_nonempty interrupt is asserted when data is left/remained in tx_fifo
```

Comment:
Shouldn't tx_empty be asserted when there is no data in tx_fifo (but is needed)?
TX FIFO should be empty at the end of a Read (by Agent), otherwise tx_nonempty interrupt is asserted.

Reply:
@igk8 thanks, I update the description